### PR TITLE
Changelogs and compatibility matrix for release 1.10.1

### PR DESCRIPTION
**Pull request by swathinsankaran**
This PR adds the changelogs for the 1.10.1 release and also updates the compatibility matrix.

### CODE REVIEW

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

**Comment by swathinsankaran** [comment]
skip jira-id-check

**Comment by swathinsankaran** [comment]
build ako

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

**Comment by swathinsankaran** [comment]
skip jira-id-check

**Comment by swathinsankaran** [comment]
build ako

**Review by DixitAakash — APPROVED** [review]
LGTM

File: docs/README.md [hunk 1]
```diff
@@ -85,13 +85,13 @@ Please refer to this [page](objects.md) for details on how AKO interprets the Ku
...
-AKO version 1.9.1 support for Kubernetes, Openshift, Avi Controller is as below:
+AKO version 1.10.1 support for Kubernetes, Openshift, Avi Controller is as below:
 
 | **Orchestrator/ Controller** | **Versions Supported** |
 | --------- | ----------- |
-| `Kubernetes` | 1.22 - 1.25 |
-| `Openshift` | 4.6 - 4.11 |
-| `Avi Controller` | 21.1.5 - 22.1.3 |
+| `Kubernetes` | 1.22 - 1.26 |
+| `Openshift` | 4.6 - 4.12 |
```
Comment (swathinsankaran) [inline docs/README.md:93]: @akshayhavile is it required to support 6 versions of openshift and 4 versions of k8s?

Comment (DixitAakash) [reply docs/README.md:93]: @swathinsankaran at least for the ako operator, we are supporting from Openshift 4.8.

File: docs/README.md [hunk 2]
```diff
@@ -85,13 +85,13 @@ Please refer to this [page](objects.md) for details on how AKO interprets the Ku
...
-AKO version 1.9.1 support for Kubernetes, Openshift, Avi Controller is as below:
+AKO version 1.10.1 support for Kubernetes, Openshift, Avi Controller is as below:
 
 | **Orchestrator/ Controller** | **Versions Supported** |
 | --------- | ----------- |
-| `Kubernetes` | 1.22 - 1.25 |
-| `Openshift` | 4.6 - 4.11 |
-| `Avi Controller` | 21.1.5 - 22.1.3 |
+| `Kubernetes` | 1.22 - 1.26 |
+| `Openshift` | 4.8 - 4.12 |
```
Comment (swathinsankaran) [inline docs/README.md:93]: @akshayhavile, @DixitAakash I have changed the openshift version to 4.8 as discussed.

Comment (swathinsankaran) [reply docs/README.md:93]: Also, 1.22 (7 months ago) and 1.23 (3 months) reached EOL.

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

**Comment by swathinsankaran** [comment]
skip jira-id-check

**Comment by swathinsankaran** [comment]
build ako

**Review by DixitAakash — APPROVED** [review]
LGTM

**Review by arihantg — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -326,4 +326,40 @@ All notable changes to this project will be documented in this file. The format
...
+ - Default parameters of an L4 and Shared VIP L4 virtual services can be changed via [`L4Rule`](docs/crds/l4rule.md) CRD.
+ - Support to configure OAuth or SAML for L7 virtual services via [`SSORule`](docs/crds/ssorule.md) CRD.
+ - AKO now supports [`OVN-Kubernetes`](docs/ovn-kubernetes.md) CNI in Openshift and [`Cilium`](docs/cilium.md) CNI in Kubernetes.
+ - Option to add [ICAP profile](docs/crds/hostrule.md#express-custom-icap-profile) to an L7 virtual service via `HostRule` CRD.
+ - Option to enable [HTTP2](docs/crds/avinfrasetting.md#custom-ports) in an L7 virtual service via `aviinfrasetting` CRD.
+ - Mixed protocol support for Shared VIP service.
+
+### Fixed
+- The VSes sometimes are not coming up and show pool placement issues when CNI is set as Calico.
+- Mixed protocol support was not working when SCTP protocol is specified with either UDP/TCP.
```
Comment (akshayhavile) [inline CHANGELOG.md:365]: need to be more specific. This is fixed for shared vip services with same port and protocol is sctp and udp/tcp. Also we should mention, L4 services with mixed protocols and same port are not supported

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

File: CHANGELOG.md [hunk 4]
```diff
@@ -326,4 +326,40 @@ All notable changes to this project will be documented in this file. The format
...
+## AKO-1.10.1
+
+### Added
+ - AKO now claims support for OCP 4.12.
+ - Default parameters of an L4 and Shared VIP L4 virtual services can be changed via [`L4Rule`](docs/crds/l4rule.md) CRD.
+ - Support to configure OAuth or SAML for L7 virtual services via [`SSORule`](docs/crds/ssorule.md) CRD.
+ - AKO now supports [`OVN-Kubernetes`](docs/ovn-kubernetes.md) CNI in Openshift and [`Cilium`](docs/cilium.md) CNI in Kubernetes.
+ - Option to add [ICAP profile](docs/crds/hostrule.md#express-custom-icap-profile) to an L7 virtual service via `HostRule` CRD.
+ - Option to enable [HTTP2](docs/crds/avinfrasetting.md#custom-ports) in an L7 virtual service via `aviinfrasetting` CRD.
+ - Mixed protocol support for Shared VIP service.
```
[the remark this answers is not in the document]
Comment (swathinsankaran) [reply CHANGELOG.md]: I have removed this.

File: CHANGELOG.md (hunk 3 above)
Comment (swathinsankaran) [reply CHANGELOG.md]: I have add the changes as per the comment.

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

**Comment by swathinsankaran** [comment]
build ako

**Comment by swathinsankaran** [comment]
skip jira-id-check

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

**Comment by swathinsankaran** [comment]
skip jira-id-check

**Comment by swathinsankaran** [comment]
build ako

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

**Comment by swathinsankaran** [comment]
skip jira-id-check

**Comment by swathinsankaran** [comment]
build ako

**Review by akshayhavile — APPROVED** [review]
Overall changes looks good. While doing public KB, we need to make sure, AVI controller compatibility mentions as For 21.x release, 21.1.5 and above and for 22.x release, 22.1.3 and above. @apalsule Please confirm this.

**Comment by swathinsankaran** [comment]
build ako

**Pull request merged**